kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/KyuubiScalaObjectMapper.scala

```diff
@@ -23,7 +23,9 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 
 class KyuubiScalaObjectMapper extends ContextResolver[ObjectMapper] {
-  private val mapper = new ObjectMapper().registerModule(DefaultScalaModule)
+  override def getContext(aClass: Class[_]): ObjectMapper = KyuubiScalaObjectMapper.mapper
+}
 
-  override def getContext(aClass: Class[_]): ObjectMapper = mapper
+object KyuubiScalaObjectMapper {
+  val mapper: ObjectMapper = new ObjectMapper().registerModule(DefaultScalaModule)
```

Comment:
you can revert the change here now.

Reply:
OK, good catch, reverted all changes in KyuubiScalaObjectMapper.